developer_manual/design/foundations.rst

```diff
@@ -0,0 +1,182 @@
+
+.. list-table::
+
+    * - .. figure:: ../images/colour-text-maxcontrast.svg
+
+           Secondary text in light theme: #767676
+
+      - .. figure:: ../images/colour-dark-theme-text-maxcontrast.svg
+
+           Secondary text in dark theme: #8C8C8C
+
+
+In addition, a softer color is used for secondary text like sublines, timestamps, and similar.
+
+
+* On web: var(--color-text-maxcontrast)
+* Android: uses default Material Design color "medium emphasis"
+* iOS: `secondaryLabel <https://developer.apple.com/documentation/uikit/uicolor/3173136-secondarylabel>`_
+* Desktop: `default Qt guidelines <https://doc.qt.io/qt-5/qpalette.html#ColorRole-enum>`_
+
+Status and indicators
+^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+
+    * - .. figure:: ../images/colour-success.svg
+
+           Success: #46BA61
+
+      - .. figure:: ../images/colour-error.svg
+
+           Error: #E9322D
+
+      - .. figure:: ../images/colour-warning.svg
+
+           Warning: #ECA700
+
+Interface elements associated with a status like success, error, or warning may also be colored to communicate the action better.
+
+While interface elements like buttons are colored differently depending on their action, the color of the text in that element is almost always either of the main text colors, that is light or dark.
+
+
+* On web:
+
+  * var(--color-success)
+  * var(--color-warning)
+  * var(--color-error)
+
+* Android: Material Design guidelines
+* iOS: `Apple HIG colors <https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/color/>`_
+
+  * success: systemGreen
+  * error: systemRed
+
+* Desktop: `default Qt guidelines <https://doc.qt.io/qt-5/qpalette.html#ColorRole-enum>`_
+
+Typography
+----------
+
+To ensure compatibility with different platforms, Nextcloud apps always use the native system font.
+
+For legibility, make sure that the text in your content uses:
+
+* **Bold** for emphasis
+* A line height between 130% and 150%
+* The default tracking of the font
+* No *italics* or UPPER CASE as these text styles are less legible
+
+The text sizes for the different platforms are:
+
+
+* Web: 15px for main text and sublines, **20px bold** for headings
+* Android: 14sp for main text, 16sp for headings
+* iOS: values from `Dynamic Type Sizes, for size Large (Default) <https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography#dynamic-type-sizes>`_
+* Desktop: `default Qt guidelines <https://doc.qt.io/qt-5/qpalette.html#ColorRole-enum>`_
+
+Icons
+-----
+
+.. image:: ../images/material-icons.png
+   :alt: Material icons
+
+Icons can be used to communicate the intent of an action, or to provide visual interest to the screen. We use monochrome icons on all platforms: `Material Design icons (default filled) <https://fonts.google.com/icons>`_ for web, Android, Windows and Linux, and `SF Symbols (default weight, scale and variant) <https://developer.apple.com/sf-symbols/>`_ for iOS and macOS.
```

Comment:
Should we specify the material design icon sizes to pass in for each use case?

Reply:
Very nice that the usage of material design icons is official now. And yes, a decision on the icon size is overdue if we want to adopt md icons Nextcloud wide. But maybe we do this in a separate issue (because it will probably result in a larger discussion and delay this PR here)?

Reply:
I think we discussed this quite a bit already, we have this comment from Jan from a recent conversation: https://github.com/nextcloud/nextcloud-vue/pull/1808#issuecomment-900998940
 I don't know if this is definitive, so @jancborchardt and @nimisha-vijay could you confirm the size at least for the web interface? This is popping up in different prs and it would be nice to have a verdict on it :)

Reply:
Yeah, good call @marcoambrosini – so we would recommend the same 20px as we use on the button component, correct?

Reply:
> 
> 
> Yeah, good call @marcoambrosini – so we would recommend the same 20px as we use on the button component, correct?

I would propose to discuss this in a separate issue, since this is quite a big decision. I created https://github.com/nextcloud/nextcloud-vue/issues/2248.